tests/deep-copy/fail_JSON.js

```diff
@@ -0,0 +1,2 @@
+const cloneDeep = (target, ...obj) =>
+  JSON.parse(JSON.stringify(Object.assign(target, ...obj)))
```

Comment:
it's great that you did a failing test for that, but I think the multiple args and all is a mistake, i'll clean it up.